src/interchange/src/avro.rs

```diff
@@ -896,16 +896,59 @@ fn build_schema(columns: &[(ColumnName, ColumnType)]) -> Schema {
             {
                 "name": "after",
                 "type": ["null", "row"]
+            },
+            {
+                "name": "transaction",
+                "type": [
+                    "null",
+                    {
+                        "name": "transaction_metadata",
+                        "type": "record",
+                        "fields": [
+                            {
+                                "name": "id",
+                                "type": "string",
+                            }
+                        ]
+                    }
+                ]
             }
         ]
     });
     Schema::parse(&schema).expect("valid schema constructed")
 }
 
+fn get_consistency_schema() -> Schema {
```

Comment:
this schema purposefully omits the `data-collections` field -- this is not hard to introduce at all it just seems extremely redundant, because each sink will have its own consistency topic and that topic will only reference one data collection

@nacrooks thoughts?

Reply:
We use the data_collections field in the timestamper code to decode Debezium records. I realise that now it's not relevant given that we can't feed in the consistency topic generated back into Materialize because of the unordered writing of data, but in the future when we can, we might want this to be a "generic" debezium topic (even with the redundant collection) so that we don't need to special-case a SINK generated one.